paparazzi-gradle-plugin/src/test/projects/multiplatform-plugin-with-android/build.gradle

```
@@ -5,7 +5,23 @@ plugins {
 }
 
 kotlin {
-  androidTarget()
+  androidTarget {
+    publishLibraryVariants('release')
+  }
+
+  sourceSets {
+    commonTest.dependencies {
+
+    }
+    commonMain.dependencies {
+
+    }
+    androidMain {
+      dependencies {
+        implementation 'androidx.appcompat:appcompat:1.7.0'
+      }
+    }
+  }
```

```suggestion
  androidTarget()
```

none of this KMP boilerplate affects the test purpose